spyder/utils/encoding.py

```diff
@@ -233,15 +239,21 @@ def write(text, filename, encoding='utf-8', mode='wb'):
     Return (eventually new) encoding
     """
     text, encoding = encode(text, encoding)
+
+    if os.name == "nt":
+        absolute_filename = to_text_string(pathlib.Path(filename).resolve())

```

Comment:
Does this work independently if the file exists or not?

If the file for some reason was "deleted"by the time this is called, is there an error we should try to handle @dalthviz ?

Reply:
Checking it seems like it can throw some errors if the path doesn't exists https://docs.python.org/3.5/library/pathlib.html#pathlib.Path.resolve

Reply:
In that case, could we add a try:except there and write a test for it?